frontend/catalyst/jax_tracer.py

```diff
@@ -1158,24 +1209,28 @@ def is_leaf(obj):
                     output = return_values_flat
                     trees = return_values_tree
 
+                _, tree_expected = jax.tree_util.tree_flatten(return_values, is_leaf=is_leaf)
                 mcm_config = qnode.execute_kwargs["mcm_config"]
                 qrp_out = trace_quantum_operations(tape, device, qreg_in, ctx, trace, mcm_config)
-                meas, meas_trees = trace_quantum_measurements(device, qrp_out, output, trees)
+
+                meas, meas_trees, out_tree_expected = trace_quantum_measurements(
```

Comment:
you can silence "unused-variable" by naming them private. What I usually do is something like `meas, _, out_tree_expected = ...`

Reply:
It is used now :)